cmake/modules/AddSwift.cmake

```diff
@@ -1473,18 +1485,15 @@ function(add_swift_library name)
         endif()
 
         set(lipo_target "${name}-${SWIFT_SDK_${sdk}_LIB_SUBDIR}")
-        _add_swift_lipo_target(
-            ${sdk}
-            ${lipo_target}
-            ${UNIVERSAL_LIBRARY_NAME}
-            ${THIN_INPUT_TARGETS})
-
         if("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin" AND SWIFTLIB_SHARED)
```

Comment:
Absolutely. I guess it doesn't affect building Android on macOS, so I didn't need to modify it. But yes, at some point it would be great to go hunting for `CMAKE_SYSTEM_NAME`s, replacing them with SDK checks!